components/mod-event/EventItem.tsx

```diff
@@ -82,17 +108,25 @@ const Report = ({
     modEvent.event.reportType === ComAtprotoModerationDefs.REASONAPPEAL
   return (
     <Card>
-      <p className="flex justify-between">
+      <div className="flex justify-between">
         <span>
           By{' '}
-          {modEvent.creatorHandle
-            ? `@${modEvent.creatorHandle}`
-            : modEvent.createdBy}
+          <LinkToAuthor
+            createdBy={modEvent.createdBy}
+            creatorHandle={modEvent.creatorHandle}
+          />
         </span>
-        {modEvent.event.reportType && !isAppeal && (
-          <ReasonBadge reasonType={modEvent.event.reportType} />
-        )}
-      </p>
+        <div>
```

Comment:
Not sure what this build issue is 🤔

Reply:
did you point at the right line? not seeing any issues here? maybe got cleaned up along the way?